docs/reference/templates.rst

```diff
@@ -196,3 +196,19 @@ specify the templates to use in the ``Admin`` service definition:
 Changes made using the ``setTemplate()`` and ``setTemplates()`` functions override the customizations
 made in the configuration file, so you can specify a global custom template and then override that
 customization on a specific ``Admin`` class.
+
+Finding configured templates
+----------------------------
+Templates that are set using the ``setTemplate()`` or ``setTemplates()`` functions can be accessed
+through the ``getTemplate($name)`` function of an Admin.
```

Comment:
"method"

Reply:
Thank you. I changed it here and in the existing previous part of the documentation.